src/RelayBoardV3.cpp

```diff
@@ -130,13 +130,28 @@ void RelayBoardV3::handle(std::shared_ptr<const pilot::Incident> value){
 
 
 void RelayBoardV3::handle(std::shared_ptr<const pilot::SystemState> value){
+	std::string system_error = "";
 	for(const auto &code : value->system_errors){
 		handle(Incident::create_ex(
 			value->time,
 			event_t::create_ex(event_type_e::ERROR, "SystemState", code.get_type_name(), code.to_string_value()),
 			0, {}, true, 3000)
 		);
+
+		if (value->system_errors.size() > 1) {
+			system_error += code.to_string_value() + ",";	
+		} else {
+			system_error = code.to_string_value();
+		}
 	}
+
+	if (system_error != "") {
+		m_system_error = system_error;
+		RCLCPP_ERROR(nh->get_logger(), "System error reported: %s", m_system_error.c_str());
```

Comment:
would it make sense to log this for once in every 5 seconds?

Reply:
No, that's what we have the message publishing for.